docs/references/strategies/languages/gradle/gradle.md

```diff
@@ -225,16 +225,12 @@ Any dependencies with following prefixes:
 - androidTest
 - debugAndroidTest
 - releaseUnitTest
+- debugUnitTest
 
 And any configuration named:
 - androidJacocoAnt
 - testApiDependenciesMetadata
 - testCompileOnlyDependenciesMetadata
-- debugUnitTestApiDependenciesMetadata
-- debugUnitTestCompileOnlyDependenciesMetadata
-- debugUnitTestImplementationDependenciesMetadata
```

Comment:
Why are we removing these other "testmetadata" configurations for the test filter?

Reply:
Ah! since, they all start with the prefix `debugUnitTest`. Now any configuration with prefix of `debugUnitTest` will not be included to classify "used deps"

Reply:
Oh that makes sense 🤦 , I was on mobile and missed that part while hoping to give a quick approval.